.eslintrc

```diff
@@ -7,10 +7,16 @@
   },
   "parser": "@babel/eslint-parser",
   "rules": {
-    "react/jsx-boolean-value": ["error","always"],
+    "react/jsx-boolean-value": 0,
```

Comment:
I think this should be set to 'never'. 
https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md